sdk/endpoints/online/sample/online-inferencing-sample.ipynb

```diff
@@ -30,7 +31,7 @@
    "outputs": [],
    "source": [
     "#get a handle to the workspace\n",
-    "ml_client = MLClient(subscription_id, resource_group, workspace)"
+    "ml_client = MLClient(InteractiveBrowserCredential(), subscription_id, resource_group, workspace)"
```

Comment:
why explicitly specify InteractiveBrowserCredential? What happens if I don't specify?

Reply:
`crendential` is now a required argument of MLCLient constructor. This aligns with other Azure SDKs and also what AK has been pushing for.

Reply:
`crendential` is now a required argument of MLCLient constructor. This aligns with other Azure SDKs and also what AK has been pushing for.

Reply:
OK. Sounds good. What other options do I have? What if i run it in a scheduled job? I cannot use interactive there.

Reply:
Yeah, you can use any credential that's supported by `azure-identity` package.  E.g. this is how you authenticate using a service principle. https://docs.microsoft.com/en-us/azure/developer/python/azure-sdk-authenticate-service-principals#authenticate-using-environment-variables